hedera-node/hedera-evm-api/src/main/java/com/hedera/services/evm/store/contracts/HederaEvmWorldStateTokenAccount.java

```diff
@@ -96,14 +99,14 @@ public boolean isEmpty() {
 
     private Code interpolatedCode() {
         if (interpolatedCode == null) {
-            final var interpolatedBytecode = proxyBytecodeFor(address);
+            final var interpolatedBytecode = bytecodeForToken(address);
             interpolatedCode =
                     Code.createLegacyCode(interpolatedBytecode, Hash.hash(interpolatedBytecode));
         }
         return interpolatedCode;
     }
 
-    public static Bytes proxyBytecodeFor(final Address address) {
+    public static Bytes bytecodeForToken(final Address address) {
```

Comment:
Probably we can return the name to `proxyBytecodeFor`, since this bytecode is really a `proxy` and it's the same for all tokens, except including their unique address.